modules/distr-tracing-tempo-config-query-frontend.adoc

```diff
@@ -8,60 +8,139 @@
 
 Query is a service that retrieves traces from storage and hosts the user interface to display them.
 
-.Parameters used by the {TempoOperator} to define Query
+.Configuration parameters for the Querier
 [options="header"]
-[cols="l, a, a, a"]
 |===
 |Parameter |Description |Values |Default value
 
-|spec:
-  query:
-    replicas:
-|Specifies the number of Query replicas to create.
-|Positive integer
+|`nodeSelector`
+|The simple form of the node-selection constraint.
+|type: object
+|
+
+|`replicas`
+|Specifies the number of replicas to be created for the component.
+|type: integer; format: int32
+|
+
+|`tolerations`
+|Specifies component-specific pod tolerations.
+|type: array
 |
 |===
 
 
-.Configuration parameters passed to Query
+.Configuration parameters for the Query Frontend
 [options="header"]
-[cols="l, a, a, a"]
 |===
 |Parameter |Description |Values |Default value
 
-|spec:
-  query:
-    options: {}
-|Configuration options that define the Query service.
+|`component`
+|Configuration of the Query Frontend component.
+|type: object
+|
+
+|`nodeSelector` (inside component)
+|The simple form of the node selection constraint.
+|type: object
+|
+
+|`replicas` (inside component)
+|Specifies the number of replicas to be created for the component.
+|type: integer; format: int32
+|
+
+|`tolerations` (inside component)
+|Specifies component-specific pod tolerations.
+|type: array
+|
+
+|`jaegerQuery`
+|Specifies the options specific to Jaeger Query.
+|type: object
+|
+
+|`enabled` (inside jaegerQuery)
+|When `enabled`, creates the Jaeger Query component,`jaegerQuery`.
+|type: boolean
+|
+
+|`ingress` (inside jaegerQuery)
+|Specifies the options for the Jaeger Query Ingress.
+|type: object
+|
+
+|`annotations` (inside jaegerQuery.ingress)
+|Specifies the annotations of the Ingress object.
+|type: object
+|
+
+|`host` (inside jaegerQuery.ingress)
+|Specifies the hostname of the Ingress object.
+|type: string
+|
+
+|`ingressClassName` (inside jaegerQuery.ingress)
+|The name of an IngressClass cluster resource. Defines which Ingress controller serves this Ingress resource.
+|type: string
 |
+
+|`route` (inside jaegerQuery.ingress)
+|Specifies OpenShift Route specific options.
```

Comment:
Suggestion: "Options for the OpenShift route"

Reply:
Thank you for the suggestion. Implementing.